.github/workflows/pr.yml

```diff
@@ -39,6 +39,28 @@ jobs:
         run: yarn build
       - name: Test
         run: yarn test
+  forge:
+    name: Forge
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [16]
```

Comment:
Is there a need to use matrix? Looks like it is only one param to use (not sure if you guys are planning to run with other nodes too). By the way node 16 is no longer maintained and considered deprecated https://nodejs.org/en/blog/announcements/nodejs16-eol

Reply:
yeah we need to move off of this but our stack currently can't automatically upgrade. @james-a-morris is working on this